packages/bruno-app/src/components/RequestTabPanel/index.js

```diff
@@ -150,7 +155,7 @@ const RequestTabPanel = () => {
       <div className="pt-4 pb-3 px-4">
         <QueryUrl item={item} collection={collection} handleRun={handleRun} />
       </div>
-      <section className="main flex flex-grow pb-4 relative">
+      <section className="main flex flex-grow pb-4 relative" onMouseDown={(e) => MiddleClickClose(e)}>
```

Comment:
Please rename `MiddleClickClose` as `handleMouseDown`

You can add a comment in the callback like this.
You also don't have to write an explicit return.
 
```javascript
const handleMouseDown = (e) => {
  // middle click
  if (e.button === 1) {
    handleCloseClick(e, focusedTab, dispatch);
  }
};
```